integration/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "mud-integration-sandbox",
```

Comment:
thoughts on renaming the integration dir to integration-sandbox to match?

Reply:
tried it out, wdyt?